vms/platformvm/cache_locked_utxos_chain_state_test.go

```diff
@@ -270,9 +326,46 @@ func TestUpdateUTXOs(t *testing.T) {
 			wantErr: false,
 			msg:     "Happy path depositing",
 		},
+		{
+			name: "Happy path depositing bonded tokens",
+			lockedStateImpl: lockedUTXOsChainStateImpl{
+				bonds: map[ids.ID]ids.Set{
+					{9, 9}: map[ids.ID]struct{}{{1, 1}: {}},
+				},
+				deposits: make(map[ids.ID]ids.Set),
+				lockedUTXOs: map[ids.ID]utxoLockState{{1, 1}: {
+					BondTxID:    &ids.ID{9, 9},
+					DepositTxID: nil,
+				}},
+				updatedUTXOs: map[ids.ID]utxoLockState{{1, 1}: {
```

Comment:
Should be nil unless your are trying to simulate versioned state with parent as some other updated versioned state

Reply:
updatedUTXOs != nil only after some update to lockedUTXOsChainStateImpl and only until Apply is called

Reply:
Thats true for all test cases

Reply:
Should it be also nil in "Unbonding" and "Undepositing" test cases?

Reply:
Ok got it, i had some concerns about this but now its clear.

Reply:
Yep, thats true for all test cases
Unless your test case is `lockedUTXOsChainStateImpl` that has some still not applied changes, `updatedUTXOs` should always be nil